addons/website/views/snippets/s_table_of_content.xml

```diff
@@ -6,7 +6,7 @@
         <div class="container">
             <div class="row s_nb_column_fixed">
                 <div class="col-lg-3 s_table_of_content_navbar_wrap s_table_of_content_navbar_sticky                         s_table_of_content_vertical_navbar d-print-none d-none d-lg-block o_not_editable o_cc o_cc1" data-name="Navbar">
-                    <div class="s_table_of_content_navbar list-group o_no_link_popover"/>
+                    <div class="s_table_of_content_navbar list-group o_no_link_popover overflow-hidden"/>
```

Comment:
![table_of_content_scrollbar](https://github.com/odoo/odoo/assets/125345965/2ea3e446-edc6-46ea-bd4d-ffb7be2089cb)

The scroll bar issue was occurring due to the negative margin set on the
[list-group-item.](https://github.com/odoo/odoo/blob/15.0/addons/web/static/lib/bootstrap/css/bootstrap.css#L5353) This negative margin caused the element to extend beyond
its container, leading to the unwanted scroll bar.
By adding the `overflow-hidden` class to the element containing these
list-group-item elements, prevent any overflow from being visible scrollbar.